internal/cmd/ping.go

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2021 Canonical Ltd
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0'
+ */
+
+package cmd
+
+import (
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/common"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	var cmd = &cobra.Command{
+		Use:   "ping",
+		Short: "A simple 'ping' endpoint that can be used as a service healthcheck",
```

Comment:
```suggestion
		Short: "The ping (health check) response for a given EdgeX microservice",
```

Reply:
As above, set this to `The ping (health check) response for EdgeX microservices`